core/src/main/java/org/apache/calcite/sql/validate/SqlValidatorImpl.java

```diff
@@ -4185,6 +4217,48 @@ private static void forEachQualified(SqlNode node, SqlValidatorScope scope,
     });
   }
 
+  /* If the supplied SqlNode when fully qualified is in the set of bypassQualifieds, then we
+  remove all entries in the qualifieds set as well as remnantMustFilterFields
```

Comment:
is this formatted correctly?

Reply:
No, it should be a javadoc comment. Also it should be written like a method javadoc, e.g. "Removes all entries..."